src/runtime/thread_pool.cc

```diff
@@ -310,9 +355,21 @@ class ThreadPool {
       }
     }
   }
+  // bind worker threads to disjoint cores
+  void SetThreadAffinity() {
+    for (int i=0; i < num_workers_; ++i) {
```

Comment:
Need to query maximum number of cores, and either wrap around or not set affinity when num_workers is bigger than number of cores

Reply:
`num_workers_` is regularized in the constructor of `ThreadPool`

Reply:
it can be set via environment variable https://github.com/yidawang/tvm/blob/06d3eceb0f9afbcb0402d55c60f531a926c0304d/src/runtime/thread_pool.cc#L276 in which case it is not regularized